src/__tests__/integration-test.ts

```diff
@@ -156,7 +158,7 @@ async function createApp<InstanceContext extends SpanContext>({
 >) {
   const app = express();
 
-  const server = new ApolloServer({
+  const server = new ApolloServer<any>({
```

Comment:
Uh yeah this is a problem, the typings for the constructor is wrong now actually. 

Will send another revision tomorrow.